hack/lib/serverless.bash

```diff
@@ -31,13 +31,9 @@ function install_serverless_previous {
 
   deploy_serverless_operator "$PREVIOUS_CSV"  || return $?
 
-  # TODO(ksuszyns): Remove this if block if no longer required
+  # TODO: Remove this when the previous release has readiness checks available (since 1.12).
   if versions.le "$(metadata.get olm.replaces)" 1.11.0; then
-    logger.info "Ensure ${SERVING_NAMESPACE} and ${EVENTING_NAMESPACE} \
-namespaces exists, as ${PREVIOUS_CSV} didn't created them automatically."
-
-    ensure_namespace "${SERVING_NAMESPACE}"
-    ensure_namespace "${EVENTING_NAMESPACE}"
+    timeout 120 "[[ \$(oc get namespace ${SERVING_NAMESPACE} ${EVENTING_NAMESPACE} ${INGRESS_NAMESPACE} --no-headers | wc -l) != 3 ]]" || return $?
```

Comment:
The Ingress Namespace is not created as part of auto-creation but actually when creating Knative Serving so this will never pass.

Reply:
Looks better now? Pushed changes...

Reply:
I'm wondering if we should also wait for knative-serving and knative-eventing namespaces in the original place.